icepyx/quest/dataset_scripts/argo.py

```diff
@@ -363,7 +361,7 @@ def _download_profile(
 
         # Consider any status other than 2xx an error
         if not resp.status_code // 100 == 2:
```

Comment:
Thinking out loud, this just jumped out at me, but it's not part of the PR. I'd like to try to use `resp.ok` instead of this pattern eventually.

Reply:
This `if` covers 200, 201, 202, 299... -- https://requests.readthedocs.io/en/latest/api/#status-code-lookup

Reply:
Oh, yeah, good point! Thanks.